From fe48a81e1620ff162390325452f632ab1bad61cb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Sat, 19 Feb 2011 18:54:39 +0000
Subject: [PATCH 01/72] gettext.h: add no-op _() and N_() wrappers
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Add a new header called gettext.h which is currently a no-op. To split
up the gettext series I'm first submitting patches to gettextize the
source tree before I add any of the Makefile and C changes needed to
actually use them.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 gettext.h |    9 +++++++++
 1 files changed, 9 insertions(+), 0 deletions(-)
 create mode 100644 gettext.h

diff --git a/gettext.h b/gettext.h
new file mode 100644
index 0000000..c68bbe9
--- /dev/null
+++ b/gettext.h
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2010 Ævar Arnfjörð Bjarmason
+ *
+ * This is a skeleton no-op implementation of gettext for Git. It'll
+ * be replaced by something that uses libintl.h and wraps gettext() in
+ * a future patch series.
+ */
+#define N_(s) (s)
+#define _(s) (s)
-- 
1.7.2.3

